### PR TITLE
Auto bower

### PR DESCRIPTION
**Pull request by ralsina**
Script to update and symlink bower dependencies (Issue #1737)

### CODE REVIEW

**Comment by Kwpolska** [comment]
It should also take care of jinja themes.

**Comment by ralsina** [comment]
I think jinjify should handle that, instead.

**Comment by ralsina** [comment]
@Kwpolska what do you think about merging this as a 1st iteration?

File: scripts/update-bower.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Update all bower packages
+bower update
+
+# Link bootstrap3 theme to bootstrap
+pushd nikola/data/themes/bootstrap3/assets/js/
+ln -sf ../../../../../../bower_components/bootstrap/dist/js/*js .
+rm npm.js
+git add *js
```
Comment (Kwpolska) [inline scripts/update-bower.sh]: `git add *` can be problematic.  use `git add .` to add everything in the current working directory.

**Comment by Kwpolska** [comment]
Change the `git add` command, add `git commit -am 'bower update'` and merge.

**Comment by Kwpolska** [comment]
Done.  I also added Jinja asset updates for the two bootstrap themes to `jinjify.py` (done via copying).  Merging.

**Pull request merged**